docs/develop/local_dev.mdx

```diff
@@ -117,10 +117,10 @@ import clsx from "clsx";
 <details>
     <summary> Wiping Restate </summary>
 
-    To start the Restate Server from a clean slate and wipe ongoing invocations and K/V state:
+    To start the Restate Server from a clean slate, stop the server then wipe ongoing invocations and K/V state:
 
     ```shell
-    restate-server --wipe all
+    rm -rf <BASE_DIR>/${HOST}
```

Comment:
Maybe replace `${HOST}` with `<NODE_NAME>`. By default `<NODE_NAME> == ${HOST}` but one can configure it to something else.